client/src/partials/journal/modals/trialBalanceMain.footer.html

```diff
@@ -0,0 +1,22 @@
+
+
+<button
+  id="cancelID"
+  type="button"
+  class="btn btn-default"
+  ng-click="TrialBalanceMainFooterCtrl.cancel()">
+    {{ "FORM.BUTTONS.CANCEL" | translate }}
+</button>
+
+<button
+  ng-if="TrialBalanceMainFooterCtrl.state.current.data.checked"
+  type="button"
+  ng-class="{
+    'btn btn-danger' : TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack.hasError,
```

Comment:
This is a huge check!  Could you alias this in the controller for clarity?  Something like
```js
vm.feedback = TrialBalanceMainFooterCtrl.state.current.data.checkingData.feedBack;
```
and then your HTML would be a nice:
```html

<button
  ng-if="TrialBalanceMainFooterCtrl.state.current.data.checked"
  type="button"
  ng-class="{
    'btn btn-danger' : feedback.hasError,
    'btn btn-warning': feedback.hasWarning,
    'btn btn-primary': feedback.hasSuccess
  }"
 >
</button>
```